core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4527,6 +4534,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.table.transform.manager.job.monitor.interval";
     public static final String TABLE_TRANSFORM_MANAGER_JOB_HISTORY_RETENTION_TIME =
         "alluxio.table.transform.manager.job.history.retention.time";
+    public static final String TABLE_METASTORE_RETRY_TIMEOUT =
```

Comment:
this is specific to the hive udb, so I think it should be part of the udb configuration.